mqtt_as/mqtt_as.py

```diff
@@ -371,6 +391,7 @@ def close(self):
         self.suback = False
         pkt = bytearray(b"\x82\0\0\0")
         self.pid = newpid(self.pid)
+        self.dprint("subscribe", topic, "pid", self.pid)
         struct.pack_into("!BH", pkt, 1, 2 + 2 + len(topic) + 1, self.pid)
         self.pkt = pkt
```

Comment:
This will overwrite self.pkt of previous subscribe, leading to a certain OSError in wait_msg when the 2nd SUBACK will be received.